java-grupa8/homework-1/tema/src/main/java8/ro/siit/tema/Main.java

```diff
@@ -0,0 +1,32 @@
+package ro.siit.tema;
+
+/**
+ * @author x-RauL-x
+ * The main class of an algorithm that creates 4 departaments of 5 SalesRepresentatives and gives the user
+ * diferent information about the departaments
+ */
+
+public class Main {
```

Comment:
This code is part of another task(homework). It should not show up in this pull request.